Add WebAssembly (Wasm) support for zoomable module.

Phaestion opened the pull request:
Hi @saket 

Thanks for an awesome library! We're developing quite a bit of Kotlin Multiplatform apps and we have a strong focus on rendering documents using KMP. Even though most of our users would typically use Android and iOS we have quite a bit of internal config tools that we run on web via WASM.

Given that your library doesn't support WASM yet we couldn't use it for our base libraries that our products and ultimately our config tools depend on.

So I endeavoured to add WASM support specifically for the `zoomable` module, it turned out to be pretty easy. The build seems to work fine for all existing modules. I built a snapshot version and published to `mavenLocal` and tested it out with one of our configuration tools, this seems to work well with the double click to zoom function.

See attached recording showing it works in browser with one of our config tools.
![Templar config tool + telephoto demo](https://github.com/user-attachments/assets/0630c300-4e1a-4a51-bed6-be3f61087033)

Phaestion committed:
Add WebAssembly (Wasm) support for zoomable module.

saket reviewed (approved):
This is great, thank you @Phaestion!

Phaestion commented:
Hi @saket. I see one screenshot test failed, but this does not look like this has anything to do with the wasm parts I added.
Is that just a flaky test? I cannot seem to run that test locally. Do you think it has something to do with the parts I added? Or is this just a fluke?

saket commented:
Yea I'm not sure what's up with that test. I was hoping to set aside some time, but life has kept me busy in the past few weeks. Let me merge this -- I'll investigate the test later.

Phaestion commented:
Thanks! I totally know what you mean @saket. Could you make a release? I would very much like to use an upstream release in our apps.

saket commented:
@Phaestion 0.16.0 is now available: https://github.com/saket/telephoto/releases/tag/0.16.0
